tests/integration/components/flexberry-datetime-picker-test.js

```diff
@@ -32,3 +32,36 @@ test('it renders', function(assert) {
   //Component does not support template block usage.
   assert.equal(this.$().text().trim(), '');
 });
+
+test('Testing dynamicProperties of flexberry-base-component', function (assert) {
+  assert.expect(2);
+
+  let propertyNameValue = 'class';
+  let propertyValueClass = 'firstClass secondClass';
+  let propertyValueClassChange = 'firstClass secondClass thirdClass';
+  let dynamicPropertiesTemp = { class: propertyValueClass, propertyName: propertyNameValue };
+
+  this.set('dynamicPropertiesTemp', dynamicPropertiesTemp);
+
+  this.render(hbs`
+    {{#flexberry-textbox
```

Comment:
You are testing 'flexberry-textbox' inside 'flexberry-date-time-picker-test.js' file.
Move into 'flexberry-textbox-test.js'.